rspec-tools/tests/test_create_rule.py

```diff
@@ -81,6 +82,57 @@ def read_counter_file(repo):
   return counter_path.read_text()
 
 
+def test_update_quickfix_status_branch(rule_creator: RuleCreator, mock_rspec_repo: Repo):
+  '''Test update_quickfix_status_branch'''
+  rule_number = '100'
+  language = 'cfamily'
+  status = 'covered'
+
+  metadata_path = Path(mock_rspec_repo.working_dir, 'rules', f'S{rule_number}', language, 'metadata.json')
+  mock_rspec_repo.git.checkout('master')
+  initial_metadata = json.loads(metadata_path.read_text())
+  assert status != initial_metadata.get('quickfix', 'unknown')
```

Comment:
I feel this assertion, as well as the one on l. 101 is too weak. As you are controlling the test inputs, you should know whether `quickfix` is present in the `initial_metadata` and `new_metadata` dictionaries, and you can assert that it is what you think it is.

Reply:
You mean, using this instead?

```suggestion
  assert status != initial_metadata.get('quickfix')
```

Reply:
That's an equivalently weak version for this case, since if the `quickfix` field is not present in `initial_metadata` and if the `quickfix` is bound to a value different from `status` the assertion would be satisfied and I am not sure which one you are asserting here.
I'd go for two assertions.
```suggestion
  assert 'quickfix' in initial_metadata
  assert status != initial_metadata['quickfix']
```

Reply:
Right, that's better indeed! I forgot that `dict.get` never raises an exception...

Reply:
Improved with 00c8e4ef9b49268715434a7280ee415ae96cf499.